.github/actions/veracode/action.yml

```diff
@@ -4,20 +4,27 @@ inputs:
   srcclr-api-token:
     description: "Agent API Token"
     required: true
-  srcclr-install-default-options:
-    description: Default options for Source Clear scan
-    required: false
-    default: '-DskipTests -Dmaven.javadoc.skip=true'
-  srcclr-install-options:
-    description: Additional options for Source Clear scan
```

Comment:
"srcclr-install-default-options" disappears, I've tested the srcclr CLI tool locally to make sure it doesn't run the tests when executing a scan, seems that it doesn't so no need for this default option.

"srcclr-install-options" Will require (described in readme) a small rework to use 
https://docs.veracode.com/r/Java_Scan_Directives#custom_maven_command
or
https://docs.veracode.com/r/Java_Scan_Directives#custom_maven_exec
To replace current behaviour for HXInsight, ESC, ATS and potentially for future Audit storage component.